scripts/flake8-config

```diff
@@ -0,0 +1,7 @@
+#
+# This file is distributed under the MIT License. See LICENSE.md for details.
+#
+
+[flake8]
+max-line-length = 100
+extend-ignore = E203
```

Comment:
I see you're ignoring `F401` in several places, shall we put it here?
People looking at the code won't have any idea of what `F401` is or why it's silenced in this file and not in another.

Reply:
Switched to `pylint` with long-form error codes